### ISSUE TITLE
Can show the dependencies version?

### ISSUE THREAD

**Issue by ChenHaoHu** [KyleBanks/depth#22]
this repo is very good! thanks!

And , can show the dependencies version?

**Comment by ec1oud** [comment]
Yes it would be very nice to see the dependency versions; ATM I'm trying to understand why a dependency (or one of its dependencies, etc.) depends on an out-of-date version of something else, the new version has removed API that it depends on, and I'm getting an `undefined` error in spite of the fact that it wants to explicitly keep using the old version that should still have the API.  So I'm guessing the same dependency gets brought in at least two ways with conflicting versions, perhaps?  If that is even possible, this tool could perhaps show it.  Maybe even highlight version conflicts with ANSI color codes, and/or highlight out-of-date version numbers.

**Comment by jammutkarsh** [comment]
Hey @ChenHaoHu,

I have implemented this feature of show dependency. Checkout my fork at [JammUtkarsh/depth](https://github.com/JammUtkarsh/depth). It's currently in the `no-internal-tree` branch.

So in that branch you will get two addational features.

1. Option to hide stdlib using `-std` flag
2. Option to show version using `-v` flag.

I would love you know the thoughts of community on this implementation.

Update: It's been [merged](https://github.com/JammUtkarsh/depth/pull/1) to my master

cc: @KyleBanks
